cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -475,13 +476,11 @@ var (
 				if devID.ApplicationID != "" {
 					device.ApplicationID = devID.ApplicationID
 				}
-				if device.SupportsJoin {
-					if devID.JoinEUI != nil {
-						device.JoinEUI = devID.JoinEUI
-					}
-					if devID.DevEUI != nil {
-						device.DevEUI = devID.DevEUI
-					}
+				if device.SupportsJoin && devID.JoinEUI != nil {
```

Comment:
IMO we should not check for `SupportsJoin` here at all. Just set whatever user has set.

FTR, 1.0.4 spec reads:
![2020-04-17-14:28:35-screenshot](https://user-images.githubusercontent.com/12877905/79569537-0e459380-80b8-11ea-8212-dda9de4f72a7.png)
The way understand it is that JoinEUI, if fact, *may* be set for ABP.

Reply:
JoinEUI has no function with ABP